ResourceNamespaceInfo.json

```diff
@@ -343,6 +343,14 @@
     "isUpdateSupportedFilter": ".*",
     "isDeleteSupportedFilter": ".*"
   },
+  {
+    "namespace": "MSGraph:DirectoryRoles",
+    "description": "Azure AD -> Roles and Administrators",
```

Comment:
"->" should be ">" just like how we have the other providers. Same applies for the below